webapp/src/pages/applicationForm/components/ApplicationForm.js

```diff
@@ -1009,14 +1010,22 @@ class ApplicationListComponent extends Component {
     }
   }
 
+  componentDidMount() {
+    eventService.getEvent(this.props.formSpec.event_id).then(response => {
+      this.setState({
+        event_type: response.event.event_type
+      });
+    })};
+  
+
   getCandidate = (allQuestions, response) => {
     const nominating_capacity = answerByQuestionKey("nominating_capacity", allQuestions, response.answers);
     if (nominating_capacity === "other") {
       let firstname = answerByQuestionKey("nomination_firstname", allQuestions, response.answers);
       let lastname = answerByQuestionKey("nomination_lastname", allQuestions, response.answers);
       return firstname + " " + lastname;
     }
-    return this.props.t("Self Nomination");
+    return  (this.state.event_type ==='JOURNAL' || this.state.event_type ==='CONTINUOUS_JOURNAL') ? this.props.t("Submission " + response.id) : this.props.t("Self Nomination");
```

Comment:
This will give you errors because we won't find the text "Submission + arbitrary numbers" in the translation.json keys (have a look in the Console output and you should see them). You'll need to rather do `this.props.t("Submission") + " " + response.id`  (ie only translate the "Submission" text). Also please add this and any new text you've added on the UI to the two translation.json files for English and French.